### PR TITLE
#161872375 Code refactor for followers

### PR DESCRIPTION
**Pull request by brucemakallan**
#### What does this PR do?
Change the JSON representation of Followers

#### Description of Task to be completed?
- Change JSON to remove the 'follower' and 'followee' dictionary keys
- Display profiles without any extra dictionary keys

#### How should this be manually tested?
GET `/api/users/3/followers`
GET `/api/users/1/following`

Both should return a list of profiles as seen below:
```
{
    "profiles": [
        {
            "id": 2,
            "username": "jackkatto",
            "first_name": "Jack",
            "last_name": "Katto",
            "bio": "Test bio"
        },
        {
            "id": 3,
            "username": "judeinno",
            "first_name": "Jude",
            "last_name": "Inno",
            "bio": "Test jude bio"
        }
    ]
}
```

#### Any background context you want to provide?
You require authentication to view followers and user following a specific user 

#### What are the relevant pivotal tracker stories?
[#161872375](https://www.pivotaltracker.com/story/show/161872375)

### CODE REVIEW

File: authors/apps/profiles/serializers.py [hunk 1]
```diff
@@ -53,45 +53,19 @@ class Meta:
...
-            "bio": profile.bio
-        }
-        return (follow_details)
+        if self.context.get("user_type") == 'followee':
+            profile = getProfileFromDatabase(
+                user_id=follow_details['followee'])
+        else:
+            profile = getProfileFromDatabase(
+                user_id=follow_details['follower'])
+        return (ProfileSerializer(profile).data)
```
Comment (judeinno) [inline authors/apps/profiles/serializers.py:67]: I like the way you refactored the serializers to use one serializer, Great job

**Review by judeinno — APPROVED** [review]
LGTM

**Review by RachaelNantale — APPROVED** [review]
LGTM

**Review by emugaya — CHANGES_REQUESTED** [review]
Please follow the advice I gave on migrations yesterday

**Pull request merged**